ra/ra.go

```diff
@@ -189,6 +190,19 @@ func NewRegistrationAuthorityImpl(
 	}, []string{"reason"})
 	stats.MustRegister(revocationReasonCounter)
 
+	authzAges := prometheus.NewHistogram(prometheus.HistogramOpts{
+		Name: "names_per_cert",
+		Help: "Histogram of the number of SANs in requested and issued certificates",
+		// authzAges keeps track of how old, in seconds, authorizations were at
+		// the time that we attached them to an order. We give it a non-standard
+		// bucket distribution so that the leftmost (closest to zero) bucket can be
+		// used exclusively for brand-new (i.e. not reused) authzs. Our buckets are:
+		// one nanosecond, one second, one minute, one hour, 7 hours (our CAA reuse
+		// time), 1 day, 2 days, 7 days, 30 days, 90 days (should be empty), +inf.
+		Buckets: []float64{0.000000001, 1, 60, 3600, 25200, 86400, 172800, 604800, 2592000, 7776000},
```

Comment:
Why no 60 day bucket? Is there an obvious reason that I'm just missing?

Reply:
Because our authorizations expire after 30 days, so in theory everything beyond that should be empty. Not even really sure why I included the 90 day bucket, to be honest...

Reply:
Ah, 🤦🏻‍♀️ that makes sense. Yeah I would honestly lose the 90 day bucket here.

Reply:
Aaaand that's already done.